services/bots/types.py

```diff
@@ -10,3 +11,19 @@
 type TokenWithOwner = Tuple[Token, Optional[Owner]]
 
 type TokenTypeMapping = Dict[TokenType, Token]
+
+
+class AdapterAuthInformation(TypedDict):
+    # This is the Authentication used with the git provider
+    token: Token
+    # token_owner is used to decide on token_refresh functions
+    token_owner: Owner | None
+    # GitHub app info - exclusive for GitHub (duh)
+    # Preferred method of authentication (if available)
+    # selected_installation_info is used to mark installations as rate-limited
```

Comment:
`selected_installation_info` means the app is rate-limited? could you rephrase this comment?